src/python/pants/backend/python/subsystems/setup.py

```diff
@@ -82,7 +82,8 @@ def register_options(cls, register):
                 "See https://pip.pypa.io/en/stable/user_guide/#constraints-files for more "
                 "information on the format of constraint files and how constraints are applied in "
                 "Pex and pip.\n\n"
-                "Mutually exclusive with `[python].enable_resolves`."
+                "Mutually exclusive with `[python].enable_resolves`, which we generally recommend "
+                "as an improvement over constraints file."
```

Comment:
Are we at a point where we're comfortable deprecating this option in favor of that one (with manual lockfile generation)?